src/volttron/messagebus/zmq/zap/zap_authenticator.py

```diff
@@ -64,13 +64,13 @@ def __init__(self, *, options: ServerOptions, credentials_store: CredentialsStor
     def is_authenticated(self, *, identity: Identity) -> bool:
```

Comment:
These are not used instead the auth service is used. not sure why I left these here